mailet/base/src/test/java/org/apache/mailet/base/test/FakeMailContext.java

```diff
@@ -44,11 +49,17 @@
         private final MailAddress sender;
         private final Collection<MailAddress> recipients;
         private final MimeMessage msg;
+        private final List<String> attributeNames;
 
-        public SentMail(MailAddress sender, Collection<MailAddress> recipients, MimeMessage msg) {
+        public SentMail(MailAddress sender, Collection<MailAddress> recipients, MimeMessage msg, List<String> attributeNames) {
             this.sender = sender;
-            this.recipients = recipients;
+            this.recipients = ImmutableList.copyOf(recipients);
```

Comment:
I am almost sure it is called only by the tests.

I had troubles with list equality. Using guava lists were appropriate and soled the issue. Hence, i remember immutable lists was a team best practice ???